File: app/src/main/java/com/harry/harrypotter/ui/fragments/WandFragment.java
package com.harry.harrypotter.ui.fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.harry.harrypotter.R;

import org.jetbrains.annotations.NotNull;

import androidx.fragment.app.Fragment;
import dagger.hilt.android.AndroidEntryPoint;

@AndroidEntryPoint
public class WandFragment extends Fragment {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
            Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_wand, container, false);
    }

    @Override
    public void onViewCreated(@NotNull View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
    }

}

File: app/src/main/java/com/harry/harrypotter/ui/main/fragments/WandFragment.kt
package com.harry.harrypotter.ui.main.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.fragment.app.activityViewModels
import com.harry.harrypotter.R
import com.harry.harrypotter.databinding.FragmentWandBinding
import com.harry.harrypotter.ui.base.BaseFragment
import com.harry.harrypotter.ui.main.vm.MainViewModel2
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class WandFragment : BaseFragment<FragmentWandBinding>() {

    override fun getViewBinding() = FragmentWandBinding.inflate(layoutInflater)

    override fun init() {

    }
}